README.md

```diff
@@ -15,6 +15,72 @@
 
 <!-- SPHINX-START -->
 
+This provides helpers for using Cython. Use:
+
+```cmake
+find_package(Cython MODULE REQUIRED VERSION 3.0)
+```
+
+If you find Python beforehand, the search will take this into account. You can
+specify a version range on CMake 3.19+. This will define a `Cython::Cython`
+target (along with a matching `CYTHON_EXECUTABLE` variable). It will also
+provide the following helper function:
+
+```cmake
+Cython_compile_pyx(<pyx_file>
+                  [LANGUAGE C | CXX]
+                  [CYTHON_ARGS <args> ...]
+                  [OUTPUT <OutputFile>]
+                  [OUTPUT_VARIABLE <OutputVariable>]
+                  )
+```
+
+This function takes a pyx file and makes a matching `.c` / `.cxx` file in the
+current binary directory (exact path can be specified with `OUTPUT`). The
+location of the produced file is placed in the variable specified by
+`OUTPUT_VARIABLE` if given. Extra arguments to the Cython executable can be
+given with `CYTHON_ARGS`, and if this is not set, it will take a default from a
+`CYTHON_ARGS` variable.
+
+If the `LANGUAGE` is not given, and both `C` and `CXX` are enabled globally,
+then the language will try to be deduced from a `# distutils: language=...`
+comment in the source file. It is an error if it cannot be deduced.
```

Comment:
I believe `cython` will default to C if no information is provided on the CLI or via a directive. I don't feel strongly that we have to match that exactly, but that would be my default choice.

Reply:
Okay